Products/CMFPlone/tests/testResourceRegistries.py

```diff
@@ -19,30 +20,32 @@
 from zope.component import getUtility
 
 
-class TestScriptsViewlet(PloneTestCase.PloneTestCase):
-    def _make_test_bundle(self):
-        registry = getUtility(IRegistry)
+def _make_test_bundle(
+    name="foobar",
+    depends="",
+):
+    registry = getUtility(IRegistry)
+
+    bundles = registry.collectionOfInterface(IBundleRegistry, prefix="plone.bundles")
+    bundle = bundles.add(name)
+    bundle.name = name
+    bundle.jscompilation = f"http://foo.bar/{name}.js"
+    bundle.csscompilation = f"http://foo.bar/{name}.css"
+    bundle.depends = depends
+    return bundle
```

Comment:
What is the reason to move this code outside of the class?

Reply:
But in the other class it is defined as a method... https://github.com/plone/Products.CMFPlone/pull/4054/files#diff-8ef40cff64bf27605c753443d198309a61a480a09ec62d66fed8152c607cb600R264

IMO leave it like it was or make the test classes inherit from a common class.